### DISCUSSION TITLE
alist安装到服务器上了，主页如何变成私密访问，凭借账号密码访问，不公开给所有人访问

### DISCUSSION THREAD

**Discussion by e5shujufenxifenxi** [AlistGo/alist#3133] [Q&A]
alist安装到腾讯云上了，已经运行正常，主页想作为家用私密资源使用，不想公开给所有人随意访问。请问如何能做到凭借账号密码登录来访问主页

**Comment by RavelloH** [comment]
在管理页面的元信息中创建以下内容即可。
![Screenshot_20230124_100957_com android chrome](https://user-images.githubusercontent.com/68409330/214198628-5e9d7f25-d7a5-4786-89f5-6cb0c69e034a.png)

同时需注意，用户页面中的guest账户不要开启`无需密码访问`权限
